packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/versions.doc.ts

```diff
@@ -268,6 +288,12 @@ Refer to the [migration guide](/docs/api/pos-ui-extensions/migrating) for more i
 - Removed in POS version: 9.31.0
 - Release day: 06/10/2024.
 
+## Important Fixes
```

Comment:
We can remove this change, since 2024-04 is not supported in the latest version of POS